src/Collapsible.js

```diff
@@ -61,12 +60,14 @@ Collapsible.propTypes = {
    * @default false
    */
   accordion: React.PropTypes.bool,
+  className: React.PropTypes.string,
+  children: React.PropTypes.node,
   /**
    * Enable popout style
    */
   popout: React.PropTypes.bool,
   defaultActiveKey: React.PropTypes.number,
-  onSelect: React.PropTypes.func,
+  onSelect: React.PropTypes.func
```

Comment:
it's a question of style... some people like double quotes... no semi-colon... and other. I agree about the diff though. I guess it's not a big deal either way :)